notebooks/1_spikesorting.ipynb

```diff
@@ -4,26 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    param_dict['num_workers'] = 4
</pre><p>you have 32 channels no? could increase this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/nwb_datajoint/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>Is it over 32 channels or the 4 electrodes?</p><p> </p>

Reply:
<p>I think <code>num_workers</code> can be set to the number of channels, which is 32 in this case.</p>

Reply:
<p>wouldn't we want to sort 4 channels at a time (I think in this case the 4 channels of the tetrode/sort group 10?), not 32, given that these are tetrode data? Anyhow, could do with a little more explanation - there is probably something I'm not following!</p>

Reply:
I don't think I'm sure about this. Maybe @khl02007 can help?

Reply:
<p>sorry I thought this was a shank of a livermore probe that has 32 channels. if this is a tetrode, the num_workers should be 4. </p><p> </p><p>by the way, it's not clear to me: are we sorting multiple tetrodes here? can we make the example sort just 1 tetrode? </p>

Reply:
We are only sorting 1 tetrode here. Sort group 10 is a single group of 4 electrodes. I think the mention of 32 unique electrode_group_names within the SortGroup section may have been the confusing thing?